### DISCUSSION TITLE
gocq为啥启动后又自动断开

### DISCUSSION THREAD

**Discussion by yllsyx** [Tsuk1ko/CQ-picfinder-robot#341] [问题求助] [answered]
![image](https://user-images.githubusercontent.com/39957638/178728154-0d48cc7b-4bed-44ed-9a41-faefb651cc12.png)
npm start时会显示online，但是回头一查进程就又变成0%了.
之前都还能运行正常，但是之后更新了一下config配置就不能用了，就算换回原配置好像也是这样。

**Comment by Tsuk1ko** [comment, marked as answer]
`npm run test` 看是什么错误

**Reply by yllsyx** [reply]
![image](https://user-images.githubusercontent.com/39957638/178745403-c0aa457e-27fb-4b21-8d5a-2c5c26887326.png)
提示配置文件错误，然后我试着改了一下文件编码成utf-8还是不行

**Reply by Miuzarte** [reply]
装一个[vsc](https://code.visualstudio.com/) 然后编辑的时候找标红的地方，你这是语法错了
![image](https://user-images.githubusercontent.com/66856838/178746534-f9ac3d5e-3e9d-49f9-a2cd-a89873646092.png)

**Reply by yllsyx** [reply]
> 
![image](https://user-images.githubusercontent.com/39957638/178747299-e186f8b4-ef52-41f3-9f58-e04919486346.png)
没显示报红

**Reply by Miuzarte** [reply]
看看右下角是不是LF
![image](https://user-images.githubusercontent.com/66856838/178748135-5c112b88-e671-4b4c-803a-5c3511b96739.png)
![F {I8CH85EFO)EV3B)%6X1Z](https://user-images.githubusercontent.com/66856838/178748019-a5cfb031-39bf-44cd-833c-b1193c8c3151.jpg)

**Reply by yllsyx** [reply]
![image](https://user-images.githubusercontent.com/39957638/178749352-9cf93f4c-f13f-46d7-a883-4ec98126454f.png)

> 看看右下角是不是LF ![image](https://user-images.githubusercontent.com/66856838/178748135-5c112b88-e671-4b4c-803a-5c3511b96739.png) ![F {I8CH85EFO)EV3B)%6X1Z](https://user-images.githubusercontent.com/66856838/178748019-a5cfb031-39bf-44cd-833c-b1193c8c3151.jpg)

![image](https://user-images.githubusercontent.com/39957638/178749381-5e9dfde5-8453-4c60-9c5e-ece99d61ef49.png)
然后变成酱紫了

**Reply by Miuzarte** [reply]
default.jsonc不要动。。。

**Reply by yllsyx** [reply]
> default.jsonc不要动。。。

嗷，确实，可以跑起来了，感谢
